recipes-security/refpolicy/refpolicy%.bbappend

```diff
@@ -0,0 +1,10 @@
+PV = "2.20210908+git${SRCPV}"
+
+SRC_URI = "git://github.com/aoscloud/refpolicy.git;protocol=https;branch=develop;name=refpolicy;destsuffix=refpolicy"
+
+SRCREV_refpolicy = "7c68e3065070b8c100503ec4a223cabf738787c1"
```

Comment:
Should be SRCREV:refpolicy according to new format.

Reply:
Done